backend/redis/utils.go

```diff
@@ -483,6 +488,30 @@ func (r *Client) GetHubspotCompanies(ctx context.Context, companies interface{})
 	return
 }
 
+func (r *Client) SetGithubRateLimitExceeded(ctx context.Context) error {
+	return r.setFlag(ctx, GithubRateLimitKey, true, time.Hour)
```

Comment:
does github have any estimates of how much the rate limit is, and how soon will it cool down? seems like an hour is a long time

Reply:
Good call! Updated this to be the reset time returned by the github api
![Screenshot 2023-08-23 at 4 49 26 PM](https://github.com/highlight/highlight/assets/17744174/95452aa1-16d3-4f80-a43f-983deb61d81d)